hadoop-common-project/hadoop-common/CHANGES.txt

```diff
@@ -2,11 +2,16 @@ Hadoop Change Log
 
 Release 2.7.2 - UNRELEASED
 
+Release 2.7.2 - 2016-01-25
```

Comment:
Nopes! You've cherry-picked into Altiscale's 2.7.1 release... So just add the line in OUR 2.7.1's Improvements section